java/plugins/jdi-elements-plugin/src/test/java/ru/sbtqa/tag/pagefactory/pages/jdielements/StepDefs.java

```diff
@@ -3,11 +3,12 @@
 import cucumber.api.java.Before;
 import ru.sbtqa.tag.pagefactory.JDIUtils;
 import ru.sbtqa.tag.pagefactory.PageFactory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'ru.sbtqa.tag.pagefactory.PageFactory'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)